doc/examples/README.md

```diff
@@ -24,3 +24,7 @@ The [`yaml2sbml`](https://github.com/yaml2sbml-dev/yaml2sbml) package translates
         *   Time-dependent right hand sides
         *   Step functions in the right hand side
         *   Function definitions
+                
+## Running the examples
+
+The notebooks come with additional dependencies, mainly for running the ODE-simulation and parameter fitting. Information on the installation of AMICI is given in its [installation guide](http://sbml.org/Special/Software/libSBML/docs/formatted/python-api/libsbml-math.html). Further dependencies ( e.g. [pyPESTO](https://github.com/ICB-DCM/pyPESTO)) can be installed via `pip install yaml2sbml[examples]`.
```

Comment:
```suggestion
The notebooks come with additional dependencies, mainly for running the ODE-simulation
and parameter fitting.
Information on the installation of AMICI is given in its
[installation guide](https://github.com/AMICI-dev/AMICI/blob/master/INSTALL.md).
Further dependencies (e.g. [pyPESTO](https://github.com/ICB-DCM/pyPESTO))
can be installed via `pip install yaml2sbml[examples]`.
```

Reply:
- No need to hyphenate "ODE-simulation"
- Could use the RTD link for the AMICI installation guide: https://amici.readthedocs.io/en/latest/python_installation.html